src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.mitxonline.Production.yaml

```diff
@@ -55,7 +55,7 @@ config:
   edxapp:framework: "earthly"
   edxapp:web_node_capacity: "9"
   edxapp:worker_node_capacity: "5"
-  edxapp:worker_instance_type: "c6a.large"
+  edxapp:worker_instance_type: "r8g.large"
```

Comment:
These are graviton instances so they'll be running ARM chips. I'm not sure if that will work for edX. Typically we go with the `a` class instances, such as `r7a.large`.